examples/low-level-api/list-nodes.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+import asyncio
+from asyncio import TimeoutError
+from datetime import datetime, timezone
+import json
+import sys
+import pathlib
+from typing import AsyncIterator, List
+
+from yapapi import props as yp
+from yapapi.log import enable_default_logger
+from yapapi.props.builder import DemandBuilder
+from yapapi.rest import Configuration, Market, Activity, Payment  # noqa
+from yapapi.rest.market import OfferProposal
+from examples import utils
+
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
```

Comment:
krunch3r here from the community, nice work!

this won't run on my windows install... i observed that from the perspective of python, the examples module is searched from its parent directory. did you mean: ```python
examples_parent_dir = pathlib.Path(__file__).resolve().parent.parent.parent
```
and to place it above line 15 or is this just extra code? ♥️

Reply:
so, i think i was looking at something else when i initially commented!

at any rate, as is,  this will emit: `ModuleNotFoundError: No module named 'examples'` on account of the import statement `from examples` preceding the module search path modification.

ironically i pulled a fix for this issue to list-offers.py some time ago and you must have been looking at another branch when you copied:
https://github.com/golemfactory/yapapi/commit/aa86ed304a284302d6934708797f432cd19069dd